auth/Auth/appsettings.json

```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "AuthPostgresConnection": "Server=localhost;Port=5432;Database=auth;Username=someuser;Password=somePassword"
+    "AuthPostgresConnection": "Server=localhost;Port=5011;Database=auth;Username=someuser;Password=somePassword"
```

Comment:
I don't understand why have you commited this, but ok

Reply:
Because this is the working configuration (on my PC, true, but my PC is the only one runnig it at the moment)